### PR TITLE
[ACORN-583] move seg-pos and parts eval to acon-py

### PR DESCRIPTION
**Pull request by ChiaraPiacentiniAGMT**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -134,8 +126,159 @@ jobs:
           name: run the integration tests
           command: |
             ~/agmt make integration-tests ARGS="-n 4"
+<<<<<<< HEAD
```
Comment (aliaugmenta) [inline .circleci/config.yml:129]: bad merge?

Comment (ChiaraPiacentiniAGMT) [reply .circleci/config.yml]: yep, sorry. Fixed it

**Review by aliaugmenta — APPROVED** [review]
👍

**Review by lawsonAGMT — APPROVED** [review]
🥳

**Pull request merged**